src/base/reconstruction.h

```diff
@@ -269,9 +272,12 @@ class Reconstruction {
   void ImportPLY(const std::vector<PlyPoint>& ply_points);
 
   // Export to other data formats.
-  bool ExportNVM(const std::string& path) const;
-  bool ExportBundler(const std::string& path,
-                     const std::string& list_path) const;
+  bool ExportNVM(const std::string& path, bool skip_distortion = false) const;
+  bool ExportCam(const std::string& path, bool skip_distortion = false) const;
```

Comment:
What is the "cam" format? What is the "Recon3D" format?

Maybe we can add a one-liner to each method here, e.g.:
```
// VisualSFM data format.
bool ExportNVM(const std::string& path, bool skip_distortion = false) const;
// Bundler data format.
bool ExportBundler(const std::string& path,	  bool ExportCam(const std::string& path, bool skip_distortion = false) const;
etc.
```

Reply:
Added detailed descriptions for new and existing formats. Let me know if that looks OK now.

Reply:
I made some minor modifications. skip_distortion also ignores distortion effects and is not just a problem with the focal length.